gremlin-core/src/test/java/org/apache/tinkerpop/gremlin/structure/io/graphson/GraphSONMapperTest.java

```diff
@@ -150,8 +162,10 @@ public void shouldHandlePath() throws Exception {
                 extend("alice", new HashSet<>(Collections.singletonList("c")));
         final String json = mapper.writeValueAsString(p);
 
-        if (version.startsWith("v1") || version.startsWith("v3") || version.startsWith("v4"))
+        if (version.startsWith("v1") || version.startsWith("v3"))
```

Comment:
do we need tests for v1 and v3?

Reply:
Probably. The old GraphSONMappers should stay as they can be used with io(). It would be best if users updated their storage format to v4, but all versions should be provided so that the user can read in an older version then write into v4.